eng/restore-toolset.sh

```diff
@@ -41,7 +41,7 @@ function InitializeCustomSDKToolset {
     return
   fi
 
-  InstallDotNetSharedFramework "1.1.2"
+  InstallDotNetSharedFramework "9.1.2"
```

Comment:
9.1.2?  That's got to be a typo.

Reply:
Already fixed in a subsequent commit. I was testing something when I accidentally commited this.